certsubmitter/certsubmitter.go

```diff
@@ -47,22 +47,18 @@ func Run(ctx context.Context, lc *client.LogClient, ca *certgen.CA, st storage.A
 		}
 
 		glog.Infof("%s: %s: adding chain...", l.URL, logStr)
-		sct, httpData, addErr := lc.AddChain(chain)
+		_, httpData, addErr := lc.AddChain(chain)
 		if len(httpData.Body) > 0 {
 			glog.Infof("%s: %s: response: %s", l.URL, logStr, httpData.Body)
 		}
 
-		// Store get-sth API call.
-		apiCall := apicall.New(ct.GetSTHStr, httpData, addErr)
+		// Store add-chain API call.
+		apiCall := apicall.New(ct.AddChainStr, httpData, addErr)
```

Comment:
Oopsie! :smile:

Reply:
Yuuup... much copy-and-pasting, not so much proof-reading :/